Q 24) A Government company claims that an average light bulb lasts 270 days. A researcher randomly selects 18 bulbs for testing. The sampled bulbs last an average of 260 days, with a standard deviation of 90 days. If the CEO's claim were true, what is the probability that 18 randomly selected bulbs would have an average life of no more than 260 days

In [1]:
from scipy import stats
from scipy.stats import norm

Population mean =270 days Sample mean = 260 days Sample SD = 90 days Sample n = 18 bulbs df = n-1 = 17<br>

Assume Null Hypothesis is: Ho = Avg life of Bulb >= 260 days<br>
Alternate Hypothesis is: Ha = Avg life of Bulb < 260 days<br>

In [2]:
# find t-scores at x=260; t=(s_mean-P_mean)/(s_SD/sqrt(n))
t=(260-270)/(90/18**0.5)
t

-0.4714045207910317

In [3]:
# Find P(X>=260) for null hypothesis

# p_value=1-stats.t.cdf(abs(t_scores),df=n-1)... Using cdf function
p_value=1-stats.t.cdf(abs(-0.4714),df=17)
p_value

0.32167411684460556

In [4]:
#  OR p_value=stats.t.sf(abs(t_score),df=n-1)... Using sf function
p_value=stats.t.sf(abs(-0.4714),df=17)
p_value

0.32167411684460556

Probability that 18 randomly selected bulbs would have an average life of no more than 260 days is <b>32.17%</b><br>Assuming significance value α = 0.05 (Standard Value)(If p_value < α ; Reject H0 and accept Ha or vice-versa)<br>
Thus, as p-value > α ; Accept H0 i.e. The CEO claims are false and the avg life of bulb > 260 days